### ISSUE TITLE
Self-service portal for being onboarded to the cluster

### ISSUE THREAD

**Issue by dtenenba** [fhdsl/FH_Cluster_Guide#36]
In a couple places you link to a Google form (which I presume sends email to scicomp?) requesting a new cluster account.
We actually have a new tool which users can use to onboard themselves. I am not sure if we are ready to announce it to the world - @chrisequalsdev can you speak to that? 

I am also concerned that a google form on the public internet could be a portal for spam.

**Comment by chrisequalsdev** [comment]
I was originally thinking that I would wait and and automate this later, however i'm now envisioning lots of requests coming in and wishing I did this first.  I started working on a new landing page for the self service portal earlier.  Current thinking is that the landing page will serve to check if a user has all the access needed to complete the Cluster 101 course and if the check fails then they can reach out to scicomp in some fashion. I should have a demo available in the next few days.
